tutorials/getting_started/things_simple.md

```diff
@@ -81,20 +84,21 @@ To find out why we can click on the `Hue Bridge`.
 ![](images/hue_bridge_config_error.png)
 
 The error description below the status gives some more information: You have to press the physical button on the device to let openHAB authenticate to it.
-Once you've done this, the Thing will become online:
+Once you've done this, the Thing will become online.
 
 ![](images/hue_bridge_online.png)
 
 ## Accept the Light Bulb Things
 
 Go back to Things.
-You will notice a new red button on that screen:
+You will notice a new red button on that screen.
 
 ![](images/hue_inbox.png)
 
-It means that there are newly discovered devices that you can add; since the Hue bridge is now working, the binding has detected all the Hue devices (bulbs, switches and so on) attached to it.
+This means that there are newly discovered devices that you can add. Since the Hue bridge is now working, the binding has detected all the Hue devices (bulbs, switches and so on) attached to it.
```

Comment:
```suggestion
This means that there are newly discovered devices that you can add.
Since the Hue bridge is now working, the binding has detected all the Hue devices (bulbs, switches and so on) attached to it.
```